src/gpu/gpu_batch_normalization_list.cpp

```diff
@@ -30,14 +30,17 @@
 #endif
 
 #if DNNL_GPU_VENDOR == DNNL_VENDOR_NVIDIA
-#include "gpu/generic/sycl/ref_batch_normalization.hpp"
 #include "gpu/nvidia/cudnn_batch_normalization.hpp"
 #endif
 
 #if DNNL_GPU_VENDOR == DNNL_VENDOR_AMD
 #include "gpu/amd/miopen_batch_normalization.hpp"
 #endif
 
+#if DNNL_GPU_VENDOR == DNNL_VENDOR_NVIDIA || DNNL_GPU_VENDOR == DNNL_VENDOR_AMD
```

Comment:
(random spot)
There is an [old pull request](https://github.com/oneapi-src/oneDNN/pull/1705), which addresses an issue that the DPC++ compiler had at the time the PR was opened. Can you confirm that the issue was fixed in the compiler and we can close that PR?

Reply:
I was not able to reproduce the issue when working on the changes for this PR. I remember I was able to reproduce it when https://github.com/oneapi-src/oneDNN/pull/1705 was opened, so I think it's safe to assume the issue has been fixed.

Reply:
Ok, let's assume it's fixed.